homeassistant/components/axion_dmx/__init__.py

```diff
@@ -0,0 +1,45 @@
+"""The Axion Lighting integration."""
+
+from __future__ import annotations
+
+from libaxion_dmx import AxionDmxApi  # Import the API class
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_LIGHT_TYPE, DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.LIGHT]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Axion Lighting from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
Please store in `entry.runtime_data`. Also extend the ConfigEntry typing

Reply:
Got it, thanks!